src/client/containers/layout.js

```diff
@@ -47,6 +47,20 @@ class Layout extends React.Component {
 		this.renderNavHeader = this.renderNavHeader.bind(this);
 	}
 
+	handleDropdownToggle(newValue) {
+		if (this._forceOpen) {
+			this.setState({menuOpen: true});
+			this._forceOpen = false;
+		}
+		else {
+			this.setState({menuOpen: newValue});
+		}
+	}
+
+	handleDropdownClick() {
+		this._forceOpen = true;
```

Comment:
When I run the code and test the ctrl+click, I get this error in the browser console:
`Uncaught TypeError: Cannot set property '_forceOpen' of undefined
    at handleDropdownClick`

This is an issue with React: the functions defined on the class do not have access to the `this` context for that class by default.
It's cumbersome, and you'll want to remember it to save yourself some looking around for the cause of an error, but here's the way to fix it: you need to bind the function to the class `this` context.
If you look in the class constructor, you'll notice we do that with other functions, in this format: `this.renderNavHeader = this.renderNavHeader.bind(this);`

Reply:
Done